static/fetch.js

```diff
@@ -1,31 +1,179 @@
 const RELOAD_TIME = 3000; 
-const PERSISTENCE_TIME = 180000;
+const PERSISTENCE_TIME_COMPLETE = 180000;
+const PERSISTENCE_TIME_PENDING = 300000;
+const PERSISTENCE_TIME_ERROR = 300000;
+
+const PrintJob = {
+    PARSE(code) {
+        for (let key in PrintJob) {
+            if (PrintJob[key].CODE === code) return PrintJob[key]
+        }
+    },
+    COMPLETED: {
+        CODE: 0,
+        ATTR: 'completed',
+        MSG: 'SUCCESS'
+    },
+    PENDING: {
+        CODE: 1,
+        ATTR: 'pending',
+        MSG: 'PENDING'
+    },
+    FILE_ERROR: {
+        CODE: 2,
+        ATTR: 'error',
+        MSG: 'FAILURE'
+    },
+    QUOTA_LIMIT: {
+        CODE: 3,
+        ATTR: 'error',
+        MSG: 'FAILURE'
+    },
+    JOB_ERROR: {
+        CODE: 4,
+        ATTR: 'error',
+        MSG: 'FAILURE'
+    }
+}
+
+
+function _element(tag, classList, attributes, innerText){
+    let temp = document.createElement(tag);
+    if (classList && !(classList instanceof Array) && typeof classList === 'string') classList = [classList];
+    else classList = [];
+    if (attributes && attributes instanceof Array) {
+        let temp = {};
```

Comment:
please give this a better name (that is different from the better name from before)